src/logplex_channel.erl

```diff
@@ -130,6 +131,13 @@ create_id() ->
             Err
     end.
 
+-spec new_id() -> id().
```

Comment:
What's the difference between the `new_id/0` and `create_id/0` functions? They both seem to do very similar things, so having two seems like a bug.